requirements-dev.in

```diff
@@ -37,7 +37,7 @@ types-python-dateutil>=2.8.3
 types-pytz==2023.3
 types-redis==4.3.20
 types-retry==0.9.9.4
-types-requests==2.26.1
+types-requests==2.31.0.6 # >= 2.31.0.7 versions require urllib>=2, which is incompatible with our dependencies
```

Comment:
Bumping this is required as otherwise `requests.exceptions.JSONDecodeError` appears as an undefined attribute (even though it exists).